spam.py

```diff
@@ -37,24 +43,28 @@ def __init__(self):
         args = utils.get_args(message)
         logger.debug(args)
         if len(args) == 0:
-            await message.edit(_("U wot? I need something to spam"))
+            ret = self.strings["need_spam"]
+            await message.edit(ret)
```

Comment:
We are migrating to utils.answer also. `utils.answer(message, self.strings["need_spam"])`